models/swin_unetr_btcv_segmentation/configs/metadata.json

```diff
@@ -19,12 +20,12 @@
         "0.1.0": "complete the model package",
         "0.0.1": "initialize the model package structure"
     },
-    "monai_version": "1.0.1",
-    "pytorch_version": "1.13.0",
-    "numpy_version": "1.21.2",
+    "monai_version": "1.2.0rc2",
+    "pytorch_version": "1.13.1",
+    "numpy_version": "1.22.4",
     "optional_packages_version": {
-        "nibabel": "3.2.1",
-        "pytorch-ignite": "0.4.8",
+        "nibabel": "4.0.1",
+        "pytorch-ignite": "0.4.9",
         "einops": "0.4.1"
     },
     "name": "Swin UNETR BTCV segmentation",
```

Comment:
Hi @tangy5 , as for #340 , if anything need to be updated, feel free to push a commit into this PR directly, thanks!

Reply:
@yiheng-wang-nv , I've updated the numbers in this PR. Thanks.